api/Window.json

```diff
@@ -1670,10 +1670,10 @@
               "version_added": "24"
             },
             "safari": {
-              "version_added": "6.1"
+              "version_added": "7"
```

Comment:
Using http://mdn-bcd-collector.appspot.com/tests/api/Window/crypto it looks like `window.crypto` was actually added in Safari 5.1. Maybe the collector can be used to update a bunch of this data?